app/dashboard/models.py

```diff
@@ -3268,7 +3268,8 @@ class HackathonEvent(SuperModel):
     logo_svg = models.FileField(blank=True)
     start_date = models.DateTimeField()
     end_date = models.DateTimeField()
-    background_color = models.CharField(max_length=255, null=True, blank=True, help_text='hexcode for the banner')
+    background_color = models.CharField(max_length=255, null=True, blank=True, help_text='hexcode for the banner, default to white')
```

Comment:
For the help text here, do we want to make a note about html colors also being allowed as in your example?

Reply:
I don't think so, just people using hex will be fine. I don't know if the user will know the entity right names also.